srcpkgs/walk/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'walk'
+pkgname=walk
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+hostmakedepends="gzip make"
+short_desc="Two utilities collectively replacing find: walk and sor"
+maintainer="Andrea D'Amore <a@d-amo.re>"
+license="Apache-2.0"
+homepage="https://github.com/bbarenblat/walk"
+distfiles="https://github.com/bbarenblat/${pkgname}/archive/refs/tags/v${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://github.com/bbarenblat/walk/archive/refs/tags/v${version}.tar.gz"
```

Reply:
Now I think you guys are just making stuff up to mess with me.

Good thing you stopped flow on this, though, since I had inadvertently reverted the `make_install_args` change (made the changes on another computer).